posts/how-to-choose-between-react-angular-or-vue.md

```diff
@@ -0,0 +1,160 @@
+
+
+### React JSX Example
+
+``` javascript
+// html is mixed in with the javascript
+function App() {
+  return (
+    <div className='App'>
+      <h1>Hello World</h1>
+    </div>
+  )
+}
+
+export default App
+```
+
+
+### Angular Brief Overview
+
+Angular is more of a heavy-weight framework being that, out of the box, it includes many built-in features you will more than likely need for a large enterprise type application. Funtionality such as routing, form validation, and testing are all included with Angular. One of it's features is it's [Command Line Interface](https://angular.io/cli) or CLI, which is a very useful tool when developing Angular projects. As a result of Angular's many included features, it has a reputation of having a steep learning curve. Angular also uses [Typescript](https://www.typescriptlang.org/) out of the box, which is known to make your code more robust and less prone to bugs. However, there's still no JSX to learn, being that Angular uses an html template which is normally in a separate file away from the core JS logic. Lastly, according to Angular's website during the time of this article publication, there aren't as many resource links for [community, forums](https://dev.to/t/angular), and [support](https://stackoverflow.com/questions/tagged/angular) on the website compared to React's website. 
+
+
+### Angular Component & Template Example
+
+``` javascript
+// component logic
+@Component({
+  selector: 'app-myApp',
+  templateUrl: './myApp.component.html',
+  styleUrls: ['./myApp.component.css']
+})
+
+export class myAppComponent implements OnInit {
+  constructor() { }
+  ngOnInit(): void { }
+}
+
+// component html template in separate file
+<p>myApp works!</p>
+```
+
+
+### Vue Overview
+
+Vue may be considered the least in popularity due to it being the newer of the frameworks and many developers or businesses haven't had the opportunity to explore the possibilities. This may have an effect on it's smallar [community](https://dev.to/t/vue) size as well. There is also a factor that Vue is built by a community of people versus an actual company like Facebook and Google, but that's open for discussion. Hence, they do have a community [forum](https://forum.vuejs.org/) as a resource link on their website.
```

Comment:
```suggestion
Vue may be considered the least popular due to it being the newer of the frameworks and many developers or businesses have not had the opportunity to explore its possibilities. This may have an effect on its smaller [community](https://dev.to/t/vue) size as well. Another factor is that Vue is community-built, compared to an actual company like Facebook and Google, but that's open for discussion. This does mean, however, that they do have a community [forum](https://forum.vuejs.org/) as a resource link on their website.
```

Reply:
The last sentence doesn't quite match the previous 2 sentences, the original intent was to say that, Vue has a small community and you can find it on their website at this link

Reply:
I changed the last sentence, what do you think?

Vue may be considered the least popular due to it being the newer of the frameworks and many developers or businesses have not had the opportunity to explore its possibilities. This may have an effect on its smaller [community](https://dev.to/t/vue) size as well. Another factor is that Vue is community-built, compared to an actual company like Facebook and Google, but that's open for discussion. You can visit Vue's community [forum](https://forum.vuejs.org/) on their website to find out more.

Reply:
I like it!  Updating my pull